String + Extensions.swift

```diff
@@ -0,0 +1,156 @@
+//
```

Comment:
this looks like a duplicate file, and is at the repo root, should probably be removed